src/rust/engine/process_executor/src/main.rs

```diff
@@ -18,6 +25,20 @@ use std::iter::Iterator;
 /// It does not perform $PATH lookup or shell expansion.
 fn main() {
   let args = App::new("process_executor")
+    .arg(
+      Arg::with_name("local-store-path")
+        .long("local-store-path")
+        .takes_value(true)
+        .required(true)
+        .help("TODO"),
+    )
+    .arg(
+      Arg::with_name("globs")
```

Comment:
This is a nice convenience, but it seems like it would be fine to require usage of `file_utils` to be used first to capture a snapshot for the case where someone wants to execute a snapshot on the contents of the current directory? Then this command would just take a digest as input.

Reply:
Done